docs/component-guidance/cta.md

```diff
@@ -0,0 +1,158 @@
+
+These are:
+
+#### Default
+
+The 'up' state that users first see.
+
+
+![Default state](https://user-images.githubusercontent.com/78355810/144259103-8900ea42-9741-4b76-9778-7195fdef08d5.png)
+
+#### Hover
+
+The 'over' state that is seen when a mouse's cursor is over the target area.
+
+![Active state](https://user-images.githubusercontent.com/78355810/144259101-979bdb19-19ae-4f26-a9b5-32ba6a874280.png)
+
+#### Active
+
+The 'pressed' state is momentarily seen on click/tap.
+
+
+![Active state](https://user-images.githubusercontent.com/78355810/144259099-35ca6ee9-2564-4cb7-8625-855015648ec5.png)
+
+#### Focus
+
+The highlighted state see when a user navigates by tabbing through on the keyboard to the CTA component.
+
+![Focus state](https://user-images.githubusercontent.com/78355810/144259100-d6cc68a5-174d-4ff3-8981-8f579ddb75e6.png)
+
+#### Loading
+
+The loading state is visually different – in contrast to the other states, it also contains animation to convey the meaning that something is in progress.
+
+
+![Loading state](https://user-images.githubusercontent.com/78355810/144259088-355b1432-ed13-4f41-bd13-e8b7d8141580.png)
+
+### Disabled
+
+The disabled state reduces the contrast and removes the click functionality of the ```cta```. 
+
+_**Accessibility note:** this state does not need to meet colour contrast ratios. The W3C states:
+“1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
+Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
+Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement._
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/144259095-ae1a9d02-0f5b-4817-bec6-b9005165967e.png)
+
+
+
+
+---
+
+## Accessible use of contrast
+
+Ensure that colours used meet or exceed WCAG AA standard for the size of text used, against its background colour. 
+
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/177174311-056405b6-0985-411b-8a5f-1da399db4cec.png)
+
+Note: AA result is for 16px text, AAA is at least 18.66px with bold weight or 24px regular.
+
+---
+
+## Content
+
+The ```cta``` component contains a link to another location (href string).
+
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/177314258-1b099114-e1eb-493a-919a-6feec39e9bef.png)
+
+
+Bi-directional text and icon position accommodates i18n (internationalisation) where some languages are read right to left.
+
+Loading message is 'false' by default but state can be changed using 'true'. The message default is "Loading..." And overwrites the CTA anonymous slot (the original CTA text) when active.
```

Comment:
Probably better to say that the default text is the token (which has a default value of "Loading...")

Reply:
updated